commit 07468a2e08023dabb9275807a9941c4bf787df68 (refs/changes/83/483/1)
Author: Andrew Hill <andrew.hill@joyent.com>
Date:   2016-09-16T09:57:47-07:00 (3 years, 1 month ago)
    
    joyent/node-manta#270 Add -p to man mput
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index b0ed749..4053dce 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -2,6 +2,8 @@
 
 ## not yet released
 
+- joyent/node-manta#270 Add -p to man mput
+
 ## 3.1.1
 
 - joyent/node-manta#261 "AssertionError: undefined (object) is required" after
diff --git a/docs/man/mput.md b/docs/man/mput.md
index eccaa77..abf24b8 100644
--- a/docs/man/mput.md
+++ b/docs/man/mput.md
@@ -72,6 +72,9 @@ OPTIONS
   When using `--file`, this switch instructs mput to first compute the MD5 of
   the file and send it in `content-md5`.
 
+`-p, --parents`
+  Create parent directories as needed.
+
 `-q, --quiet`
   Do not display a progress meter.
 
diff --git a/man/man1/mput.1 b/man/man1/mput.1
index bc8e41a..d05265a 100644
--- a/man/man1/mput.1
+++ b/man/man1/mput.1
@@ -73,6 +73,9 @@ either be in \fB\fC~/.ssh\fR or loaded in the SSH agent via \fB\fCssh\-add\fR\&.
 When using \fB\fC\-\-file\fR, this switch instructs mput to first compute the MD5 of
 the file and send it in \fB\fCcontent\-md5\fR\&.
 .TP
+\fB\fC\-p, \-\-parents\fR
+Create parent directories as needed.
+.TP
 \fB\fC\-q, \-\-quiet\fR
 Do not display a progress meter.
 .TP
@@ -127,5 +130,4 @@ $ mput \-vv ~~/stor/foo 2>&1 | bunyan
 .PP
 DSA keys do not work when loaded via the SSH agent.
 .PP
-Report bugs at Github
-\[la]https://github.com/joyent/node-manta/issues\[ra]
+Report bugs at Github \[la]https://github.com/joyent/node-manta/issues\[ra]
